website/client/store/actions/shops.js

```diff
@@ -41,6 +41,14 @@ async function buyArmoire (store, params) {
   const quantity = params.quantity || 1;
   let armoire = content.armoire;
 
+  buyOp(store.state.user.data, {
+    params: {
+      key: 'armoire',
+    },
+    type: 'armoire',
+    quantity,
+  });
+
```

Comment:
@esbena This code had been removed in https://github.com/HabitRPG/habitica/pull/9463 and I'm guessing it was added back here inadvertently when fixing merge conflicts. In a couple of days I'm going to remove it again to fix https://github.com/HabitRPG/habitica/issues/9432 but let us know if there's a reason to not do that (apart from https://github.com/HabitRPG/habitica/issues/9527 which is a less damaging bug than #9432).